src/server/helpers/entityRouteUtils.js

```diff
@@ -84,8 +84,8 @@ export function generateEntityProps(
 		action,
 		entityType,
 		heading: isEdit ?
-			`Edit ${entityName}` :
-			`Create ${entityName}`,
+			`Edit ${entityName} : ${entity.defaultAlias.name}` :
```

Comment:
Let's switch them around so that if you have 1000 tabs open it will be easier to tell them apart:
`Edit ${entity.defaultAlias.name} (${entityName})`